toolbox

```diff
@@ -1679,39 +1692,41 @@ update_container_and_image_names()
 
 usage()
 {
-    echo "Usage: toolbox [-v | --verbose]"
-    echo "               [-y | --assumeyes]"
-    echo "               create [--candidate-registry]"
-    echo "                      [-c | --container <name>]"
-    echo "                      [-i | --image <name>]"
-    echo "                      [-r | --release <release>]"
-    echo "   or: toolbox [-v | --verbose]"
-    echo "               [-y | --assumeyes]"
-    echo "               enter [-c | --container <name>]"
-    echo "                     [-r | --release <release>]"
-    echo "   or: toolbox [-v | --verbose]"
-    echo "               [-y | --assumeyes]"
-    echo "               init-container --home"
-    echo "                              --home-link"
-    echo "                              --monitor-host"
-    echo "                              --shell"
-    echo "                              --uid"
-    echo "                              --user"
-    echo "   or: toolbox [-v | --verbose]"
-    echo "               [-y | --assumeyes]"
-    echo "               list [-c | --containers]"
-    echo "                    [-i | --images]"
-    echo "   or: toolbox [-y | --assumeyes]"
-    echo "               rm [-a | --all]"
-    echo "                  [-f | --force] [<container> ...]"
-    echo "   or: toolbox [-y | --assumeyes]"
-    echo "               rmi [-a | --all]"
-    echo "                   [-f | --force] [<image> ...]"
-    echo "   or: toolbox [-v | --verbose]"
-    echo "               [-y | --assumeyes]"
-    echo "               run [-c | --container <name>]"
-    echo "                   [-r | --release <release>] <command>"
-    echo "   or: toolbox --help"
+    echo "Usage: toolbox [arguments] COMMAND [options]"
```

Comment:
Yes, this format is definitely better than before! :)

Although, I wonder if we should also convert the `--help` output to show the `toolbox(1)` manual page? Piping the output from `man(1)` works fine (eg., `man ls | cat` and such) so it wouldn't hinder people from grepping the output for whatever reason.

It's tempting to not have to maintain the documentation across multiple places, but maybe there are drawbacks I haven't thought about.

What do you think?

Reply:
It would simplify the maintenance for sure. And it shouldn't be too hard to implement.

If we were to use the output of man how should we format it? The same way it is now?? Or should we make it more git-like? Just the synopsis, all the commands + quick description and possible arguments?

Reply:
I meant that `toolbox --help` should be identical to `toolbox help`. ie., just run `man toolbox`. I am sorry for not being more clear. I see that you put in a ton of effort to reformat the manual. Sorry about that. :(

I don't quite see the usefulness of the raw textual format for showing usage information. It's not paged and there's no markup, so the format is more rudimentary as compared to the manual; and things like `toolbox help create | wc -l` work, so the fancier manual doesn't really seem to prevent any text processing.